libraries/net/node.cpp

```diff
@@ -3400,14 +3410,36 @@ namespace graphene { namespace net { namespace detail {
         }
         catch ( const fc::exception& e )
         {
-          wlog( "client rejected message sent by peer ${peer}, ${e}", ("peer", originating_peer->get_remote_endpoint() )("e", e) );
+          switch( e.code() )
+          {
+          // log common exceptions in debug level
+          case 3030001: // duplicate_transaction
+          case 3050101: // limit_order_create_kill_unfilled
+          case 3050102: // limit_order_create_market_not_whitelisted
+          case 3050103: // limit_order_create_market_blacklisted
+          case 3050104: // limit_order_create_selling_asset_unauthorized
+          case 3050105: // limit_order_create_receiving_asset_unauthorized
+          case 3050106: // limit_order_create_insufficient_balance
+          case 3050201: // limit_order_cancel_nonexist_order
+          case 3050202: // limit_order_cancel_owner_mismatch
```

Comment:
I feel this code (a hard-coded list of magic numbers) is a bit ugly. Any idea to improve?

Reply:
Does using e. g. `limit_order_cancel_nonexist_order::code_enum::code_value` work?

Reply:
Thanks, it works.

But I still think hard-coding a list here is not very good, since we may need to add more items  later. It would be better if we can make it somewhat configurable, e.g. can define the log level when throwing the exception, or use a config file, etc. Perhaps not worth the efforts though.